plugins/modules/net_tools/nmcli.py

```diff
@@ -95,6 +95,11 @@
             - Set metric level of ipv4 routes configured on interface.
         type: int
         version_added: 2.0.0
+    routing_rules4:
+        description:
+            - Is the same as in an C(ip route add) command, except always requires specifying a priority.
+        type: str
+        version_added: 3.2.0
```

Comment:
Unfortunately 3.2.0 has been released and this PR didn't make it for that.
```suggestion
        version_added: 3.3.0
```